marklogic-client-api/src/main/java/com/marklogic/client/dataservices/impl/CallManagerImpl.java

```diff
@@ -607,9 +607,9 @@ private R callImpl(DatabaseClient client, CallArgs args) {
   interface EventedCaller<E extends CallEvent> extends EndpointDefiner {
     E callForEvent(DatabaseClient client, CallArgs args) throws Exception;
   }
-  static abstract class CallerImpl<E extends CallEvent> extends EndpointDefinerImpl implements EventedCaller<E> {
+  public static abstract class CallerImpl<E extends CallEvent> extends EndpointDefinerImpl implements EventedCaller<E> {
```

Comment:
Again, if we test the public interfaces, can we keep the implementation package local?